README.md

```diff
@@ -1,20 +1,122 @@
+
+### Create an Azure subscription
+Create an Azure workspace by using the [configuration notebook](https://github.com/Azure/MachineLearningNotebooks/blob/master/configuration.ipynb)
+
+### Install the required packages
+In order to install `azureml-responsibleai` package you will need a python virtual environment. You can create a python virtual environment using `conda`.
+```c
+conda create -n azureml_env python=3.8
+activate azureml_env
+```
+
+After activating your environment, if this is your first time running the RAI Dashboard in private preview then continue to the [setup instructions](https://github.com/Azure/RAI-vNext-Preview/blob/mergePrivatePreview_MAD/docs/Setup.md) to do a one-time setup for your workspace.
+
+
+
+### Generating Responsibleai AI Dashboard insights
+Once you have created an Azure workspace and registered your components in the one-time setup above, you can create a Responsible AI dashboard via the CLI or SDK. Start here for `examples` [folder](examples) to get started.
+
+### Viewing your Responsible AI Dashboard in the AzureML studio portal
+After generating the Responsible AI insights, you can view them in your associated workspace in AzureML studio, under your model registry.
+
+![01](images/01_model_registry.png)
+1. Go to your model registry in your AzureML studio workspace
+2. Click on the model you've uploaded your Responsible AI insights for
+
+![02](images/02_model_details.png)
+3. Click on the tab for `Responsible AI dashboard (preview)` under your model details page
+
+![03](images/03_responsibleaitoolbox.png)
+4. Under the `Responsible AI dashboard (preview)` tab of your model details, you will see a list of your uploaded Responsible AI insights. You can upload more than one Responsible AI dashboards for each model. Each row represents one dashboard, with information on which components were uploaded to each dashboard (i.e. explanations, counterfactuals, etc).
+
+![04](images/04_dashboard.png)
+5. At anytime viewing the dashboard, if you wish to return to the model details page, click on `Back to model details`
+<ol type="A">
+  <li>You can view the dashboard insights for each component filtered down on a global cohort you specify. Hovering over the global cohort name will show the number of datapoints and filters in that cohort as a tooltip.</li>
+  <li>Switch which global cohort you are applying to the dashboard.</li>
+  <li>Create a new global cohort based on filters you can apply in a flyout panel.</li>
+  <li>View a list of all global cohorts created and duplicate, edit or delete them.</li>
+  <li>View a list of all Responsible AI components you've uploaded to this dashboard as well as deleting components. The layout of the dashboard will reflect the order of the components in this list.</li>
```

Comment:
Not so much _deleting_ as _hiding for this session_ surely?

Reply:
Yes you can delete cohorts from the cohorts settings panel

Reply:
To me "deleting" implies that it's gone for good, rather than just gone for the current session.

Reply:
Yes it is gone for good

Reply:
Since we do not save it anywhere else

Reply:
It can't be gone for good. The uploaded run artifacts are immutable.